new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in feature1 branch.
```

Comment:
Good that you decided to add this line of code.

Reply:
Thanks for the comment, Jukol.